### ISSUE TITLE
Expose atttypid

### ISSUE THREAD

**Issue by kurayama** [2ndQuadrant/pglogical#75]
Are there any plans in the near future to expose the attributes `atttypid` (oid) when sending the table definition on pg_output?

**Comment by promag** [comment]
@ringerc care to give feedback?

**Comment by ringerc** [comment]
Yes, we'd like to, and IIRC the protocol document defines how it'll be done.

It's necessary to allow binary replication to be safe and robust.

It's not roadmapped and we don't plan to work on it soon. Patches would be welcome if this functionality is important to you. We work on customer priorities first, and current customer priorities are elsewhere.
